setup.py

```diff
@@ -25,7 +25,7 @@
 from os.path import join as pjoin
 
 
-version = '0.61.0'
+version = '0.62.1'
```

Comment:
Why `.1`?

Reply:
I have bumped to 0.70.0 given the multiple changes. I can even see a case for a change in the major version.